### DISCUSSION TITLE
Really low FPS on Android version

### DISCUSSION THREAD

**Discussion by roxandtol** [ppy/osu#20129] [Q&A] [answered]
Hey, I'm running Lazer on my phone (Realme 8i, a mid tier phone), and even though the screen is 120Hz, i barely get 40fps in some std beatmaps, usually in stream maps, in jump beatmaps, i barely get 25fps. I also noticed a lot of lag spikes

**Comment by smoogipoo** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Especially on android devices, where the hardware can be of varying quality, it may be impossible to run well on some devices. Please be patient and wait for future optimisations.
